src/components/Spinner.tsx

```diff
@@ -0,0 +1,25 @@
+const Spinner = () => {
```

Comment:
Are you rewriting and duplicating this?
Why not reuse it from the same place ?

Remove all the svgs and images we don't need 
I see additional that aren't necessary for this PR

Reply:
- its a reusable component 
- [x] remove unnecessary images